notebooks/a01_manifold_class.ipynb

```diff
@@ -0,0 +1,1630 @@
+{
```

Comment:
<p>NIT:  f:(x1,...,xd)→f(x1,..., xd)</p><p> </p><p>Typo: because</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geomstats/geomstats/pull/1340/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>done</p>